### ISSUE TITLE
Feature Request: extra Stress

### ISSUE THREAD

**Issue by SmokeRaven667** [justinross/foundry-bitd-alternate-sheets#83]
So, if Shadows crew takes Steady upgrade, each scoundrel gets +1 stress box.
We thought we would remember. We didn't and everyone got a TRAUMA when maybe they should not have.
Anyhow, I realize the mechanics of looking at Crew upgrades to update the PC sheets would be problematic, but how about ability to click a plus next to stress teeth to add another one on a per-sheet basis?

**Comment by justinross** [comment]
Yeah, checking crew upgrades would be an ordeal, but theoretically, you could add an item to each character with an effect on it that adds 1 to their max stress.

I’ll go try to find the actual field name

**Comment by justinross** [comment]
Okay so, you can make a new item and add an effect to it like this:
![IMG_2583](https://github.com/justinross/foundry-bitd-alternate-sheets/assets/1120106/87c6e0ea-6bf8-44cc-989f-60a98a790454)

Then any characters you add that item to will have one extra stress.

**Comment by ImproperSubset** [comment]
I think this may be addressed by PR #106 available in rel-1.0.16. If a scoundrel's associated crew has Steady from the system module, alt-sheet should now update the stress on the alt character sheet. The system module's Steady active effect on the crew gets inherited by the scoundrels. You may have to close the character sheet and reopen it to get the update?

The alt-sheet doesn't have a mechanism to make the crew association. But, you can do it if you flip over to the default sheet, make the association, and the flip back to alt-sheet.
